darwin/future/meta/objects/stage.py

```diff
@@ -74,8 +74,10 @@ def move_attached_files_to_stage(self, new_stage_id: UUID) -> Stage:
             self.meta_params["workflow_id"],
             self.meta_params["dataset_id"],
         )
-        ids = [x.id for x in self.item_ids.collect_all()]
-        move_items_to_stage(self.client, slug, w_id, d_id, new_stage_id, ids)
+        ids = [str(x.id) for x in self.item_ids.collect_all()]
+        move_items_to_stage(
+            self.client, slug, w_id, d_id, new_stage_id, {"item_ids": ids}
```

Comment:
I know these aren't yours, but non-descriptive variable names could be renamed.

Reply:
Good catch, have amended